### PR TITLE
another report option to collect links between hitchwiki spot descritions and the corresponding hitchmap spot

### PR DESCRIPTION
**Pull request by tillwenke**
review after #73

### CODE REVIEW

**Comment by tillwenke** [comment]
odd behavior, could not figure out why: after viewing a spot that has a hitchwiki link, this link displays as well for all other spots even if they do not have a hitchwiki link. @bopjesvla do you have an idea why this happens?

**Comment by bderui** [comment]
Commented on issue

**Comment by bopjesvla** [comment]
I don't know, but ideally we'd do the linking automatically. Many Hitchwiki articles have lat lon information we can parse. The database is freely available.

**Comment by bopjesvla** [comment]
Amsterdam has coordinates: `<map lat='52.38278242102423' lng='4.90844612121582' zoom='11' view='3'/> `

Ideally, we'd provide this map.

**Comment by bopjesvla** [comment]
And the other way around: download the Hitchwiki db dump and extract all of the coordinates. Then, for every spot near one of those coordinates, link the page.

**Comment by tillwenke** [comment]
This does not reflect what I am trying to achieve.

There are specific spots that are described in the hitchwiki. usually for a city there are main directions. each direction can have various options. like https://hitchwiki.org/en/Dresden#Option_1 - this mentions 2 particular spots that (in this case) are kindly liked to hitchmap spots.
So there is a whole high quality detailed description of those spots on hitchwiki that we currently do not make available to hitchmap users.
this pr is a first step towards leveraging this information and connecting map and wiki.

File: modify_db.ipynb
Comment (bopjesvla) [inline modify_db.ipynb:1]: are you sure this is necessary? doesn't the pandas to_sql create the table if it doesn't exist?

**Review by bopjesvla — CHANGES_REQUESTED** [review]

**Pull request merged**